src/main/webapp/WEB-INF/views/cars/all.jsp

```diff
@@ -0,0 +1,39 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All cars</title>
+    <style type="text/css">
+        TABLE {
+            width: 300px;
+            border-collapse: collapse;
+        }
+
+        TD,TR {
+            padding: 3px;
+            border: 1px solid black;
```

Comment:
You repeat this code. I suggest creating a separate style.css file and just adding it.
For example: 
<style>
    <% @ include file = "/ WEB-INF / style.css"%>
</style>